api/v1alpha1/dataset.go

```diff
@@ -29,5 +29,6 @@ var (
 	// LabelDatasetScene defines the content type of this dataset
 	LabelDatasetContentType = Group + "/content-type"
 	// LabelDatasetBestCase defines the best case to use this dataset
-	LabelDatasetBestCase = Group + "/best-case"
+	LabelDatasetBestCase  = Group + "/best-case"
+	LabelDatasetFinalizer = Group + "/finalizers"
```

Comment:
we have a common finalizer label at https://github.com/kubeagi/arcadia/blob/main/api/v1alpha1/common.go#L25